sdk/table/azure-table/README.md

```diff
@@ -199,7 +199,7 @@ Querying entities in the table
 ```python
 from azure.table import TableClient
 
-table = TableClient.from_connection_string(conn_str="<connection_string>", table_name="my_table")
+table = TableClient.from_connection_string(conn_str="<connection_string>", table_name="mytable")
 entity = table.query_entities(results_per_page=3)
```

Comment:
pretty sure `query_entities` takes a required positional parameter `filter`. @LibbaLawrence can confirm.

Or maybe we want this sample to demonstrate `list_entities`? In which case we should update the method name.